### PR TITLE
8321958: @param/@return descriptions of ZoneRules#isDaylightSavings() are incorrect

### PR DESCRIPTION
**Pull request by naotoj**
Small document correction on a return description.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change requires CSR request [JDK-8322044](https://bugs.openjdk.org/browse/JDK-8322044) to be approved

### Issues
 * [JDK-8321958](https://bugs.openjdk.org/browse/JDK-8321958): @param/@return descriptions of ZoneRules#isDaylightSavings() are incorrect (**Bug** - P4)
 * [JDK-8322044](https://bugs.openjdk.org/browse/JDK-8322044): @param/@return descriptions of ZoneRules#isDaylightSavings() are incorrect (**CSR**)

### Reviewers
 * [Justin Lu](https://openjdk.org/census#jlu) (@justin-curtis-lu - Committer) ⚠️ Review applies to [1025c8a0](https://git.openjdk.org/jdk/pull/17098/files/1025c8a0fbada443cb0f402370c7b2d02677eb48)
 * [Joe Wang](https://openjdk.org/census#joehw) (@JoeWang-Java - **Reviewer**) ⚠️ Review applies to [1025c8a0](https://git.openjdk.org/jdk/pull/17098/files/1025c8a0fbada443cb0f402370c7b2d02677eb48)
 * [Jaikiran Pai](https://openjdk.org/census#jpai) (@jaikiran - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/17098/head:pull/17098` \
`$ git checkout pull/17098`

Update a local copy of the PR: \
`$ git checkout pull/17098` \
`$ git pull https://git.openjdk.org/jdk.git pull/17098/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 17098`

View PR using the GUI difftool: \
`$ git pr show -t 17098`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/17098.diff">https://git.openjdk.org/jdk/pull/17098.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/17098#issuecomment-1854841140)

### CODE REVIEW

**Review by justin-curtis-lu — APPROVED** [review]

**Review by JoeWang-Java — APPROVED** [review]

File: src/java.base/share/classes/java/time/zone/ZoneRules.java [hunk 1]
```diff
@@ -830,7 +830,7 @@ public Duration getDaylightSavings(Instant instant) {
      *
      * @param instant  the instant to find the offset information for, not null, but null
```
Comment (jaikiran) [inline src/java.base/share/classes/java/time/zone/ZoneRules.java:831]: Hello Naoto, do you think this `@param` description too will have to be reworded? It seems to be a copy/paste of the `getDaylightSavings()` method.

Comment (naotoj) [reply src/java.base/share/classes/java/time/zone/ZoneRules.java]: Hi Jai, looks like it was copied/pasted from `getStandardOffset()`, but you are right, the description needs to be corrected.

**Review by jaikiran — APPROVED** [review]
Thank you for the update Naoto. The current change looks good to me.

**Comment by naotoj** [comment]
Thanks for the reviews!

/integrate

**Pull request closed without merging**